src/app/shared/search/search-result.component.ts

```diff
@@ -157,4 +157,14 @@ export class SearchResultComponent implements OnChanges {
     }
     return isAscending ? SortOrder.ASCENDING : isDescending ? SortOrder.DESCENDING : SortOrder.UNSORTED;
   }
+
+  getFirstResult(): number {
+    let currentPage = (this.selected.page ? this.selected.page : 1);
```

Comment:
const would be a better option instead of let as this is not modified with in the scope

Reply:
Done.